src/styles/index.scss

```diff
@@ -1,3 +1,206 @@
+@import 'resetsettings';
+@import 'variables';
+
 body {
   margin: 0;
+  font-family: Roboto, sans-serif;
+  box-sizing: border-box;
+}
+
+* {
+  box-sizing: border-box;
+}
+
+.header {
+  display: flex;
+  justify-content: space-between;
+  align-items: center;
+  padding: 0 50px;
+  box-shadow: 0 2px 4px 0 #0000000d;
+
+  &__logo {
+    display: flex;
+  }
+}
+
+.nav {
+  &__list {
+    display: flex;
+    gap: 20px;
+  }
+
+  &__link {
+    display: flex;
+    align-items: center;
+    justify-content: center;
+    height: 60px;
+
+    font-size: 12px;
+    line-height: 14px;
+    font-weight: 500;
+    text-transform: uppercase;
+    color: #060b35;
+
+    &:hover {
+      color: $color-blue-accent;
+    }
+  }
+}
+
+.is-active {
+  position: relative;
+
+  color: $color-blue-accent;
+
+  &::after {
+    content: '';
+    position: absolute;
+    bottom: 0;
+    left: 0;
+    right: 0;
+
+    height: 4px;
+
+    background-color: $color-blue-accent;
+    border-radius: 8px;
+  }
+}
+
+.card {
+  width: $card-width;
+  padding: 32px 16px 16px;
+
+  border: 1px solid #f3f3f3;
+  border-radius: 5px;
+
+  &__photo {
+    display: block;
+
+    margin: 0 auto 40px;
+    width: 160px;
+    height: 134px;
+  }
+
+  &__title {
+    margin-bottom: 4px;
+
+    font-size: 12px;
+    line-height: 18px;
+    font-weight: 500;
+    color: $color-main-accent;
+    &:hover {
+      color: #34568b;
+    }
+  }
+  
+  &__code {
+    margin-bottom: 16px;
+
+    font-size: 10px;
+    line-height: 14px;
+    color: $color-gray;
+  }
+
+  &__rating {
+    display: flex;
+    align-items: center;
+    justify-content: space-between;
+
+    margin-bottom: 24px;
+  }
+
+  &__reviews {
+    font-size: 10px;
+    line-height: 14px;
+    color: $color-main-accent;
+  }
+
+  &__price {
+    display: flex;
+    justify-content: space-between;
+    align-items: center;
+
+    margin-bottom: 16px;
+
+    &-title {
+      font-size: 12px;
+      line-height: 18px;
+      color: $color-gray;
+    }
+
+    &-value {
+      font-size: 16px;
+      line-height: 18px;
+      font-weight: 700;
+      color: $color-main-accent;
+    }
+  }
+
+  &__button {
+    display: flex;
+    justify-content: center;
+    align-items: center;
+
+    height: 40px;
+
+    font-size: 14px;
+    line-height: 16px;
+    font-weight: 700;
+    color: #fff;
+
+    background-color: $color-blue-accent;
+    border-radius: 5px;
+
+    transition:
+      background-color $transition-duration,
+      color $transition-duration,
+      border-color $transition-duration;
+
+    &:hover {
+      background-color: #fff;
+      color: $color-blue-accent;
+      border: 1px solid #00acdc;
+    }
+  }
+}
```

Comment:
This is about zooming in on the card with a hover
![image](https://github.com/mate-academy/layout_catalog/assets/54860641/b226da68-c95f-4d51-8e81-b5f99e94af3d)


```suggestion
    transition:
      background-color $transition-duration,
      color $transition-duration,
      border-color $transition-duration,
      transform $transition-duration;

    &:hover {
      background-color: #fff;
      color: $color-blue-accent;
      border: 1px solid #00acdc;
      transform: scale(120%);
    }
  }
}
```